packages/core/database/src/query/helpers/order-by.ts

```diff
@@ -8,7 +9,10 @@ import type { Ctx } from '../types';
 
 type OrderByCtx = Ctx & { alias?: string };
```

Comment:
I thought Ctx already contained an alias, if it's the case there might be conflicts between the 2, worth using a diff name if that's the case 🤔  (

Reply:
I didn't touch this one, I'll leave it for now :)